### ISSUE TITLE
Formatting Removed at Line Break in Audience View

### ISSUE THREAD

**Issue by CameronDevine** [TheoTech/spf.io#5]
When line breaks are added to the audience view any markdown formatting is not rendered. An example of this would be a long line of text,

```
We continue in our sermon series **Partners in the Gospel as we have study Paul’s letter to the Philippian Church**.
```
If spf.io decides to break this line after the word study, the result should be,

> We continue in our sermon series **Partners in the Gospel as we have study**
> **Paul’s letter to the Philippian Church**.

However with the current behavior it will create the following result,

> We continue in our sermon series **Partners in the Gospel as we have study**
> Paul’s letter to the Philippian Church.

**Comment by chrislim** [comment]
The example you mentioned should be addressed in the next release.

**Comment by chrislim** [comment]
The next release is in prod.

**Comment by CameronDevine** [comment]
I noticed an issue with this during our service today. It appears that shorter lines of text retain the raw markdown formatting while longer lines which get split are formatted perfectly.

For example,

```markdown
**So, what about us today?**
```

renders in the audience view as

\*\*So, what about us today?\*\*

while

```markdown
**Importantly, how can we as the body of Christ–the Church–stand together and support one another in faithful living in the world?**
```

renders as

**Importantly, how can we as the body of Christ–the Church–stand together and support one another in faithful living in the world?**

This is what I noticed during the service today, but I haven't had a chance to test this more extensively, so I could be wrong about this.
